tailwind.config.js

```diff
@@ -16,6 +16,16 @@ const fontSizes = {
   '5xl': '2.5rem',
 };
 module.exports = {
+  purge: {
+    enabled: process.env.NODE_ENV === 'production',
```

Comment:
I _think_ Vue CLI takes care of enabling this for you in production already, so I don't think you need this line.

Reply:
(Might be wrong)

Reply:
yes you're right i did come across this, it does by default. after some digging, think the issue is that the tailwind extractor does not include mdx ext.

will try your above suggestion with seeing which command is running